src/django_components/component.py

```diff
@@ -1960,61 +1999,60 @@ def render_to_response(
         slots: Optional[Any] = None,
         escape_slots_content: bool = True,
         type: RenderType = "document",
+        render_dependencies: bool = True,
         request: Optional[HttpRequest] = None,
-        *response_args: Any,
         **response_kwargs: Any,
     ) -> HttpResponse:
         """
-        Render the component and wrap the content in the response class.
-
-        The response class is taken from `Component.response_class`. Defaults to `django.http.HttpResponse`.
-
-        This is the interface for the `django.views.View` class which allows us to
-        use components as Django views with `component.as_view()`.
-
-        Inputs:
-
-        - `args` - Positional args for the component. This is the same as calling the component
-          as `{% component "my_comp" arg1 arg2 ... %}`
-        - `kwargs` - Kwargs for the component. This is the same as calling the component
-          as `{% component "my_comp" key1=val1 key2=val2 ... %}`
-        - `slots` - Component slot fills. This is the same as pasing `{% fill %}` tags to the component.
-            Accepts a dictionary of `{ slot_name: slot_content }` where `slot_content` can be a string
-            or render function.
-        - `escape_slots_content` - Whether the content from `slots` should be escaped.
-        - `context` - A context (dictionary or Django's Context) within which the component
-          is rendered. The keys on the context can be accessed from within the template.
-            - NOTE: In "isolated" mode, context is NOT accessible, and data MUST be passed via
-              component's args and kwargs.
-        - `type` - Configure how to handle JS and CSS dependencies.
-            - `"document"` (default) - JS dependencies are inserted into `{% component_js_dependencies %}`,
-              or to the end of the `<body>` tag. CSS dependencies are inserted into
-              `{% component_css_dependencies %}`, or the end of the `<head>` tag.
-            - `"fragment"` - `{% component_js_dependencies %}` and `{% component_css_dependencies %}`,
-              are ignored, and a script that loads the JS and CSS dependencies is appended to the HTML.
-        - `request` - The request object. This is only required when needing to use RequestContext,
-                      e.g. to enable template `context_processors`.
-
-        Any additional args and kwargs are passed to the `response_class`.
-
-        Example:
-        ```py
-        MyComponent.render_to_response(
-            args=[1, "two", {}],
+        Render the component and wrap the content in an HTTP response class.
```

Comment:
The `render()` and `render_to_response()` methods are finally documented, as is now most of the Component class API 🚀